python/fledge/services/core/server.py

```diff
@@ -1237,6 +1280,18 @@ async def get_auth_token(cls, request: web.Request) -> web.Response:
             :Example:
                 curl -sX GET http://localhost:<core mgt port>/fledge/service/authtoken?_vid=<>
         """
+        bearer_token = request.headers.get('bearer_token', "")
```

Comment:
we should handle the Auth token via request header here the same way, we did in last commits. Eventually this endpoint will go away as we had this interim solution for (foglamp-management) poll agent service. and that will be updated to use the originally planned mechanism with (single use) token arg and return JWT token on register.

Reply:
This is related to "Poll service"

Reply:
Add a Jira for this. We should remove this `request.headers.get('bearer_token', "")` thing anyway and use Authorization header. Until then poll agent can continue to work on old mechanism using `_vid`/ explicit verification_id